build.sbt

```diff
@@ -72,14 +72,16 @@ lazy val cats = project
   .settings(catsSettings)
   .dependsOn(dataset % "test->test;compile->compile;provided->provided")
 
-lazy val `cats-spark31` = project
+lazy val `cats-spark31` = (project in file("cats"))
   .settings(name := "frameless-cats-spark31")
+  .settings(target := file("cats-spark31/target"))
```

Comment:
The idea is to take sources from the common project folder but to put artifacts into a separate one in order not to create binaries hell.

Reply:
Nope, we will have projects in their own folders through sources are going to point to the common project.